CMakeLists.txt

```diff
@@ -110,15 +111,19 @@ else()
   include_directories(${CMAKE_CURRENT_BINARY_DIR})
 endif()
 
-enable_testing()
-
-include(googletest)
+if (TESTING)
+  enable_testing()
+  find_package(GTest)
```

Comment:
Could you please remove find_package(GTest), it's unrelated to the patch
also this lib needs relatively new gtest, so installed version may not work